plugins/package-managers/node/src/main/kotlin/pnpm/Pnpm.kt

```diff
@@ -54,7 +54,7 @@ internal object PnpmCommand : CommandLineTool {
 @OrtPlugin(
     id = "PNPM",
     displayName = "PNPM",
-    description = "The PNPM package manager for Node.js.",
+    description = "The PNPM package manager for JavaScript.",
```

Comment:
Should we align the other way around? The "N" in PNPM (and NPM) stands for "Node" after all.

Reply:
Why?